Add docs about managing scorecard  using terraform (#462)
port-labs/port-docs: docs/promote-scorecards/tutorial.md

Therefore the level of the entity is `Bronze` because it passed all the rules in the `Bronze` level (hasSlackChannel)

### From Terraform

> In order to create a scorecard from the Terraform provider, you will need to use the `port_scorecard` resource.

Reviewer: Maybe add a link to the proper terraform provider docs. when clicking on the word Terraform

> Here is an example of how to create the Ownership scorecard from the Terraform provider:

Reviewer: ```suggestion
Here is an example of how to create an Ownership scorecard with the Terraform provider:
```

The request body will include the existing body of the Scorecard, after the desired updates to the existing Scorecard have been applied.

### From Terraform

> In order to update a scorecard from the Terraform provider, you will need to run the `terraform apply -target=port_scorecard.<resourceId>` command with the updated scorecard resource.

Reviewer: ```suggestion
In order to update a scorecard with the Terraform provider, you will need to run the `terraform apply -target=port_scorecard.<resourceId>` command with the updated scorecard resource.
```

:::note
When using the multiple update Scorecards `https://api.getport.io/v1/blueprints/{blueprint_identifier}/scorecards` PUT request, keep in mind that you will see a new `id` property. This is used via Port to identify the Scorecard in order to be able to update its properties
:::

:::note

> Delete Scorecard from Terraform is by using the `terraform destroy -target=port_scorecard.<resourceId>` command with the scorecard resource you want to delete.

Reviewer: Instead of putting this inside a note block, add sub headings to the delete section (similar to how you have create - from the api, from terraform).

> Delete Scorecard from Terraform is by using the `terraform destroy -target=port_scorecard.<resourceId>` command with the scorecard resource you want to delete.

Reviewer: ```suggestion
In order to delete a scorecard using the Terraform provider, use the `terraform destroy -target=port_scorecard.<resourceId>` command with the scorecard resource you want to delete. (remember that it is also possible to remove the definition of the `port_scorecard` resource from the `.tf` file and run `terraform apply`)
```
